### PR TITLE
DOCS-10241: Improve X-RateLimit headers explanation

### PR DESCRIPTION
**Pull request by mtqp**

### CODE REVIEW

File: modules/ROOT/pages/rate-limiting-sla-policy.adoc [hunk 1]
```diff
@@ -193,7 +193,7 @@ Each response header has information about the current state of the request:
 * X-Ratelimit-Limit: The maximum available requests per window 
 * X-Ratelimit-Reset: The remaining time, in milliseconds, until a new window starts
 
-MuleSoft advises that you expose these headers only if the API is used within the organization. In a public API scenario, this data can be used maliciously to invoke spikes in traffic that leave your API without any quota left.
+By default, the X-RateLimit headers are disabled in the response. You can enable these headers by selecting `Expose Headers` when you configure the policy.
```
Comment (dukesphere) [inline modules/ROOT/pages/rate-limiting-sla-policy.adoc]: Is "Expose Headers" in the UI (to be bold font `*Expose Headers*`) or a code-like with back ticks, as shown above?

File: modules/ROOT/pages/rate-limiting.adoc [hunk 2]
```diff
@@ -218,8 +218,7 @@ Each response header has information about the current state of the request:
 * X-Ratelimit-Limit: The maximum available requests per window
 * X-Ratelimit-Reset: The remaining time, in milliseconds, until a new window starts
 
-MuleSoft advises that you expose these headers only if the API is used within the organization. In a public API scenario, this data can be used maliciously to invoke spikes in traffic that leaves your API without any quota left.
-
+By default, the X-RateLimit headers are disabled in the response. You can enable these headers by selecting `Expose Headers` when you configure the policy.
```
Comment (dukesphere) [inline modules/ROOT/pages/rate-limiting.adoc]: same as above re Expose Headers

File: modules/ROOT/pages/rate-limiting.adoc [hunk 3]
```diff
@@ -235,4 +234,4 @@ Use Rate Limiting and Rate-Limiting SLA policies for accountability and to enfor
 
 * xref:tutorial-manage-an-api.adoc[Applying a Policy and SLA Tier]
 * xref:delete-sla-tier-task.adoc[Removing SLA Tiers from API Manager]
-* xref:resource-level-policies-about.adoc[Reviewing Resource Level Policies]
\ No newline at end of file
+* xref:resource-level-policies-about.adoc[Reviewing Resource Level Policies]
```
Comment (dukesphere) [inline modules/ROOT/pages/rate-limiting.adoc]: Should it be Resource-Level or Resource-level (i.e., with a hyphen)?

File: modules/ROOT/pages/spike-control-reference.adoc [hunk 4]
```diff
@@ -109,9 +109,9 @@ Each response header has information about the current state of the request:
 * X-Ratelimit-Limit: The maximum available requests per window
 * X-Ratelimit-Reset: The remaining time, in milliseconds, until the oldest request ages enough to fall outside of the sliding window. 
 +
-If quota is still available, then this header is set to 0 (zero), becuase the algorithm can still serve new requests.
+If quota is still available, then this header is set to 0 (zero), because the algorithm can still serve new requests.
```
Comment (dukesphere) [inline modules/ROOT/pages/spike-control-reference.adoc]: comma not needed before because

File: modules/ROOT/pages/spike-control-reference.adoc [hunk 5]
```diff
@@ -109,9 +109,9 @@ Each response header has information about the current state of the request:
...
 
-Expose these headers only if the API is used within the organization. For public APIs, this data can be used maliciously to invoke spikes in traffic that leaves your API without any quota left.
+By default, the X-RateLimit headers are disabled in the response. You can enable these headers by selecting `Expose Headers` when you configure the policy.
```
Comment (dukesphere) [inline modules/ROOT/pages/spike-control-reference.adoc]: same re Expose Headers

**Review by dukesphere — DISMISSED** [review]
approved but with style questions

File: modules/ROOT/pages/rate-limiting-sla-policy.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/rate-limiting-sla-policy.adoc]: Done! Thanks!

File: modules/ROOT/pages/spike-control-reference.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/pages/spike-control-reference.adoc]: Done! Thanks!

File: modules/ROOT/pages/spike-control-reference.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/spike-control-reference.adoc]: Done! Thanks!

**Review by dukesphere — APPROVED** [review]
looks good!

**Pull request merged**